src/index.html

```diff
@@ -16,17 +16,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <form
+      class="search-bar search-bar-big"
+      data-qa="big"
+    >
+      <label class="search-bar__label search-bar__lable-big">
```

Comment:
There is a typo in the class name 'search-bar__lable-big'. It should be 'search-bar__label-big'.